src/stepFunctions/asl/client.ts

```diff
@@ -83,8 +84,8 @@ export async function activate(extensionContext: ExtensionContext) {
         },
         synchronize: {
             // Synchronize the setting section 'json' to the server
-            configurationSection: ['asl'],
-            fileEvents: workspace.createFileSystemWatcher('**/*.{asl.json,asl}'),
+            configurationSection: ['asl', 'yasl'],
+            fileEvents: workspace.createFileSystemWatcher('**/*.{asl.json,asl.yml,asl.yaml,asl}'),
```

Comment:
If `.asl` is a valid extension, should `.yasl` be one as well?

Reply:
nit: keep asj.json and asl together, just like asl.yml and asl.yaml should be together.

I was also wondering about yasl earlier in the review

Reply:
I don't think there is a plan to have 'yasl' as a valid extension ending yet